src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoData.cs

```diff
@@ -41,10 +41,66 @@ public static string GetAbbreviatedEraName(CultureInfo cultureInfo)
             throw GetCultureNotSupportedException(cultureInfo);
         }
 
+        public static string GetLongDatePattern(CultureInfo cultureInfo)
+        {
+            if (string.Equals(cultureInfo.Name, "en-US", StringComparison.OrdinalIgnoreCase))
+            {
+                return s_isWindows ? "dddd, MMMM dd, yyyy" : "dddd, MMMM d, yyyy";
+            }
+            if (string.Equals(cultureInfo.Name, "es-ES", StringComparison.OrdinalIgnoreCase))
+            {
+                return s_isWindows ? "dddd, d' de 'MMMM' de 'yyyy" : "dddd, d 'de' MMMM 'de' yyyy";
+            }
+
+            throw GetCultureNotSupportedException(cultureInfo);
+        }
+
+        public static string GetMonthDayPattern(CultureInfo cultureInfo)
+        {
+            if (string.Equals(cultureInfo.Name, "en-US", StringComparison.OrdinalIgnoreCase))
+            {
+                return "MMMM d";
+            }
+            if (string.Equals(cultureInfo.Name, "es-ES", StringComparison.OrdinalIgnoreCase))
+            {
+                return s_isWindows ? "d' de 'MMMM" : "d 'de' MMMM";
+            }
+
+            throw GetCultureNotSupportedException(cultureInfo);
+        }
+
+        public static string GetShortDatePattern(CultureInfo cultureInfo)
+        {
+            if (string.Equals(cultureInfo.Name, "en-US", StringComparison.OrdinalIgnoreCase))
+            {
+                return "M/d/yyyy";
+            }
+            if (string.Equals(cultureInfo.Name, "es-ES", StringComparison.OrdinalIgnoreCase))
+            {
+                return s_isWindows ? "dd/MM/yyyy" : "d/M/yyyy";
+            }
+
+            throw GetCultureNotSupportedException(cultureInfo);
+        }
+
+        public static string GetYearMonthPattern(CultureInfo cultureInfo)
+        {
+            if (string.Equals(cultureInfo.Name, "en-US", StringComparison.OrdinalIgnoreCase))
+            {
+                return s_isWindows ? "MMMM, yyyy" : "MMMM yyyy";
+            }
+            if (string.Equals(cultureInfo.Name, "es-ES", StringComparison.OrdinalIgnoreCase))
+            {
+                return s_isWindows ? "MMMM' de 'yyyy" : "MMMM 'de' yyyy";
+            }
+
+            throw GetCultureNotSupportedException(cultureInfo);
+        }
+
         private static Exception GetCultureNotSupportedException(CultureInfo cultureInfo)
         {
-            return new NotSupportedException(string.Format("The culture '{0}' with calendar '{1}' is not supported.", 
-                cultureInfo.Name, 
+            return new NotSupportedException(string.Format("The culture '{0}' with calendar '{1}' is not supported.",
```

Comment:
I'm not sure that fits.  From [MSDN](https://msdn.microsoft.com/en-us/library/system.globalization.culturenotfoundexception(v=vs.110).aspx): The exception that is thrown when a method is invoked which attempts to construct a culture that is not available on the machine.

In this case, the culture isn't supported by the method, which means you should add the culture to the test data.

But this whole file will be reverted because all the data will be in lined, per your other comment.